autoload/lsp/ui/vim.vim

```diff
@@ -410,25 +410,141 @@ function! s:apply_workspace_edits(workspace_edits) abort
     endif
 endfunction
 
+
 function! s:apply_text_edits(uri, text_edits) abort
+    " https://microsoft.github.io/language-server-protocol/specification#textedit
+    " The order in the array defines the order in which the inserted string
```

Comment:
This is qualified in the spec with "If multiple inserts have the same position"

I don't think you can count edits without the same position already being ordered correctly (though I suspect many servers do that anyway)

Reply:
Yeah you are right, we should probably sort the the edits based on its lines as you did on: vim-lsc. 

What you think @prabirshrestha ?

Reply:
I agree that those text_edits should be sorted. Note that we should use stable sort and sort by starting range position. Please take a look at my PR: https://github.com/prabirshrestha/vim-lsp/pull/174/commits/be5e876dd3faf5571c349fd8c73bca3ebb45ed11, and consider just copying the sort/compare methods and adding them here.